stripe-core/api/stripe-core.api

```diff
@@ -88,8 +88,8 @@ public final class com/stripe/android/core/exception/APIConnectionException$Comp
 
 public final class com/stripe/android/core/exception/APIException : com/stripe/android/core/exception/StripeException {
 	public fun <init> ()V
-	public fun <init> (Lcom/stripe/android/core/StripeError;Ljava/lang/String;ILjava/lang/String;Ljava/lang/Throwable;)V
-	public synthetic fun <init> (Lcom/stripe/android/core/StripeError;Ljava/lang/String;ILjava/lang/String;Ljava/lang/Throwable;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public fun <init> (Lcom/stripe/android/core/StripeError;Ljava/lang/String;Ljava/lang/Integer;Ljava/lang/String;Ljava/lang/Throwable;)V
```

Comment:
Hmmmmm.... This might be a blocker. I don't think we can change this. It's a breaking API change :/

Reply:
😭 ok I'll retire this PR then. Here's a different PR which just doesn't log a status code of 0: https://github.com/stripe/stripe-android/pull/8229